packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -2,22 +2,25 @@
 
 ## Unreleased
 
+## 1.56.0 - (January 12, 2024)
+
+* Changed
+  * Minor dependency version bump.
```

Comment:
This doesn't look right to me either... this folder tree entry should be under here https://github.com/cerner/terra-framework/pull/1981/files#diff-382fcbe2f7c7da08636573af3dcd3cfc1f05427e93c260d61fde217696eed522R22, and maybe some of the others as well

Reply:
It looks like terra-folder-tree docs were released 2 days ago here: https://github.com/cerner/terra-framework/pull/1976. So it might be just a version bump.

Reply:
Update: the fix: https://github.com/cerner/terra-framework/pull/1981/commits/1c4958a7914bd033a957666d51e0a239093a2259